spec/mailers/shipment_mailer_spec.rb

```diff
@@ -44,4 +58,10 @@
     shipment_email = Spree::ShipmentMailer.shipped_email(shipment)
     expect(shipment_email.body).to include("Your order from #{distributor.name} has been shipped")
   end
+
+  it "picked_up email includes different text in body" do
+    text = "Your order from #{distributor.name} has been picked-up"
```

Comment:
If you set the distributor name then you don't need to use this variable and the output is more predictable.

Reply:
The distributor's name keeps changing, and I couldn't figure out how to set it so I've left it as is.

I've addressed other comments, please have a look and if it's good to go I'll sqash and push. 

Thank You for the review Maikel :smile: